docs/troubleshoot/security.md

```diff
@@ -60,21 +63,21 @@ A keyless signature is published per image. The signature is uploaded to the Rek
 
 ```shell
 cosign verify \
-  --certificate-identity "https://github.com/ratify-project/ratify/.github/workflows/publish-dev-assets.yml@refs/heads/dev" \
+  --certificate-identity-regexp "https://github.com/ratify-project/ratify/.github/workflows/publish-package.yml@*" \
   --certificate-oidc-issuer https://token.actions.githubusercontent.com \
   --certificate-github-workflow-repository ratify-project/ratify \
-  ghcr.io/ratify-project/ratify-dev:latest
+  ghcr.io/ratify-project/ratify:v1.4.0
```

Comment:
The purpose of the commands and their output in the doc is to show an example of how to verify supply chain artifacts. More of a sample then an up to date of the latest release. If we feel this should be up to date with latest release we will have to wait until v1.4.0 image is release. SInce we don't have v1.4.0, the outputs of the commands for docker build attestations will not match an image that actually exists. 

Gatekeeper's guidance does not keep the commands/outputs up to date with latest release either: https://open-policy-agent.github.io/gatekeeper/website/docs/security

Reply:
@susanshi I have verified locally that all the dev image commands are still functional